samples/matter/lock/README.rst

```diff
@@ -73,6 +73,27 @@ For details, see the `Commissioning the device`_ section.
 
 .. matter_door_lock_sample_remote_testing_end
 
+Door lock credentials
+=====================
+By default the application supports only PIN code credentials.
+Additionally, it is possible to implement support for other door lock credential types with the use of ``CredentialsManager`` module.
```

Comment:
```suggestion
=====================

By default, the application supports only PIN code credentials, but it is possible to implement support for other door lock credential types by using the ``CredentialsManager`` module.
```
Should the rest of the description in this section be made more generic? Alternatively, I can try to reshuffle the first few sentences so that it's clear that the rest of the information is specifically about the default implementation.